our-team/guide-our-team.md

```diff
@@ -0,0 +1,159 @@
+Ideally, your community moderator team should be equipped with adequate training and authority to effectively respond to Code of Conduct incidents.
+See the [Code of Conduct template](../code-of-conduct) for more resources.
+
+In this section, you can describe what tasks your community moderators are responsible for, adapting the provided boilerplate text for your needs.
+
+Then, list the names of the individuals who are currently on this team.
+If possible, indicate the best way to contact these individuals.
+Remember to keep their personal contact information private and only provide project-level methods of communication if possible.
+
+At the end of this section, link to your Code of Conduct and your Code of Conduct response documents so that contributors can know how to work with this team.
+
+
+### About the "Name of first team or role" section
+
+This section provides some placeholder text for one of your teams.
+
+In this section:
+
+1. Change the heading to the name of the team you want to describe.
+2. Include a sentence or two that explains the purpose of this team.
+3. List the specific tasks this team works on.
+4. List the names of the individuals who are currently on this team. If possible, indicate the best way to contact these individuals. Remember to keep their personal contact information private and only provide project-level methods of communication if possible.
+
+Listing the name of the specific role these individuals have is optional.
+
+You could also include the role for some individuals and not others.
+
+For example:
+
+- Chair: Dorothy Gale, [Slack handle](https://example.slack.com/team/U0123)
+- Vice-chair: Frank Baum, [Slack handle](https://example.slack.com/team/U0456)
+- Zeb Hugson, [Slack handle](https://example.slack.com/team/U0789)
+
+At the end of this section, link to any documents that can help people know how to contact this team or how to join the team.
+You can include any other helpful documents as needed.
+
+
+### About the "Name of second team or role" section
+
+This section is a placeholder text for another team.
+Fill it out identically to how you filled out the previous team.
+
+You can add additional teams as needed.
+
+
+### About the "Thank you to past team members" section
+
+This section is optional.
+
+If you would like to list past members and thank them for their service, you can list those individuals here.
+
+
+
+## Implementation and maintenance strategy
+
+Even if you're just a team of one or two people, it can help to add this page to your repository or your project's website.
+
+After you add this document to your project, set a calendar appointment to review it on a regular schedule.
+Alternatively, you can simply make a mental note to update it any time a new individual is added or removed from a team.
+
+
+
+## Acknowledgements
+
+While creating this template, the authors were inspired by the (https://www.writethedocs.org/team/)[Write the Docs Team] page.

```

Comment:
todo (blocking): Link syntax should be `[Write the Docs Team]((https://www.writethedocs.org/team/)`